src/formatDate.js

```diff
@@ -50,7 +50,24 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const from = Array.from(fromFormat);
+  const to = Array.from(toFormat);
+  const sep = to.pop();
+  const tempOBJ = {};
```

Comment:
I would change the name of this constant as well so it is clear what is it for.

Reply:
done :)